common/error.h

```diff
@@ -67,7 +67,7 @@ class [[nodiscard]] Error : public Printable<Error> {
 // Holds a value of type `T`, or an Error explaining why the value is
 // unavailable.
 template <typename T>
-class [[nodiscard]] ErrorOr {
```

Comment:
Optional, no change needed: I lean toward keeping `[[nodiscard]]` here, although it's less clear-cut than with `Error`. The risk of the error being discarded is much lower, but it's not zero, and the noise cost of a class-level attribute is much lower than a function-level attribute.

Reply:
As above, keeping.